arc_solvers/nn/util.py

```diff
@@ -50,7 +50,7 @@ def seq2vec_seq_aggregate(seq_tensor, mask, aggregate, bidirectional, dim=1):
     seq_tensor_masked = seq_tensor * mask.unsqueeze(-1)
     aggr_func = None
     if aggregate == "last":
-        raise NotImplemented("This is currently not supported.")
+        raise NotImplemented("This is currently not supported with AllenNLP 0.2.")
```

Comment:
Looks like this is not a problem right now, but we really need to update this repo to the latest AllenNLP. If you are doing any workarounds for the old version, please let me know before you spend any time on it. I'd rather just update the dependency.

Reply:
Ideally the numbers shouldn't be effected by changing version numbers. But if they are being effected, I would rather not update this public repo which is meant to reproduce our reported numbers. But that doesn't sound like a good long-term solution either. Maybe we create a branch for the current (older) version and link that branch from the leaderboard for the older solvers.

Reply:
I tried to upgrade to the new AllenNLP but there were some compatibility issues of the old models and reverted the changes. There were some operations that changed that would require retraining the models, etc..